src/FakeItEasy/Creation/DelegateProxies/DelegateProxyGenerator.cs

```diff
@@ -163,7 +163,7 @@ private static NewArrayExpression WrapParametersInObjectArray(ParameterExpressio
 
         private class DelegateCallInterceptedEventRaiser
         {
-            public static readonly MethodInfo RaiseMethod = typeof(DelegateCallInterceptedEventRaiser).GetMethod("Raise");
+            public static readonly MethodInfo RaiseMethod = typeof(DelegateCallInterceptedEventRaiser).GetMethod(nameof(Raise));

```

Comment:
No, it refers to the method, which is closer in scope (just checked). If we want to be extra careful, we could fully qualify it (`DelegateCallInterceptedEventRaiser.Raise`).